src/master-playlist-controller.js

```diff
@@ -446,6 +446,22 @@ export default class MasterPlaylistController extends videojs.EventTarget {
     this.audioPlaylistLoader_.start();
   }
 
+
+  /**
+   * Re-tune playback quality level for the current player
+   * conditions. This method may perform destructive actions, like
+   * removing already buffered content, to readjust the currently
+   * active playlist quickly.
+   */
+  fastQualityChange_() {
```

Comment:
The underscore means "don't use outside this project" to me. I feel like this is a handy debugging capability but I don't think we advertise it as a first-class part of the API (today).